boards/pyruler/4.x/code.py

```diff
@@ -78,23 +79,54 @@ def type_alt_code(code):
         leds[i].value = c
     if caps[0]:
         if ENABLE_KEYBOARD:
-            if WINDOWS_COMPUTER:
+            if WINDOWS:
                 type_alt_code(234)
-            else:
+            elif MAC:
                 kbd.send(Keycode.ALT, Keycode.Z)
+            else:
+                kbd.press(Keycode.CONTROL)
+                kbd.press(Keycode.SHIFT)
+                kbd.press(Keycode.U)
+                kbd.release_all()
```

Comment:
You can rewrite this as `kbd.press(Keycode.CONTROL, Keycode.SHIFT, Keycode.U)`, and similarly for the combination ones.

Reply:
I initially did combine those keypresses. It worked on Linux but not Chrome OS. The 'u' keypress needs to follow the modifiers. This would work.
`kbd.press(Keycode.CONTROL, Keycode.SHIFT)`
`kbd.press(Keycode.U)`

Reply:
Oh, that's weird: I'm not sure why they are requiring them to follow.

Reply:
Good thing you tested on something else besides vanilla Linux!

Reply:
I actually meant to say do `kbd.send(..., ...)` which also does the appropriate `release`, but that's moot now.

Reply:
Combined the modifier keys in the Linux blocks.